code/game/objects/items/weapons/mine.dm

```diff
@@ -110,6 +110,7 @@
 			anchored = TRUE
 			armed = TRUE
 			update_icon()
+			log_admin("[user]([user.ckey]) have placed [src] at ([user.x],[user.y],[user.z]).")
```

Comment:
```suggestion
			log_admin("[key_name(user)] has placed [src] at ([x],[y],[z]).")
```
It may be better to log the position of the mine in case in the future a special mine could be placed not exactly on top of the user.